asset/index.html

```diff
@@ -0,0 +1,404 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+    <meta http-equiv="x-ua-compatible" content="ie=edge"/>
+    <title>Plexus</title>
+    <!-- MDB icon -->
+    <link rel="icon" href="static/img/mdb-favicon.ico" type="image/x-icon"/>
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.2/css/all.css"/>
+    <!-- Google Fonts Roboto -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"/>
+    <!-- MDB -->
+    <link rel="stylesheet" href="static/css/mdb.min.css"/>
+
+    <style>
+        textarea {
+            resize: none;
+        }
+
+        .text-box {
+            margin: 0 0 1rem 0;
+            min-width: 480px;
+        }
+
+        .text-container {
+            margin: 0 0 1rem 0;
+            white-space: nowrap;
+            position: relative;
+        }
+
+        .text-container .text-icon {
+            position: absolute;
+            bottom: 2%;
+            right: 0.5%;
+        }
+
+        .label-on-border {
+            position: absolute;
+            top: -1rem;
+            padding: 0 5px;
+            background: white;
+        }
+
+        .div-border {
+            border: 1px solid;
+            padding: 1rem;
+            border-radius: 5px;
+        }
+
+        .fa-copy {
+            cursor: pointer;
+        }
+
+    </style>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script>
+        function copy(id) {
+            navigator.clipboard.writeText($(id).val())
+        }
+
+        function clearSessionDetails() {
+            $('#created-session').html('');
+        }
+
+        function clearSessionCreated() {
+            $('#session-details').html('');
+        }
+
+        function getSession(agentMSH, sessionURL) {
+            // console.log(agentMSH)
+            let dHTML = '<h2 class="mt-5 mb-3">Session details</h2>'
```

Comment:
If no IE11 support is needed (See ticket) then [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) should be used here.